lib/common/model.js

```diff
@@ -553,6 +652,113 @@ function ModelFactory (
                 $pullAll: values
             };
             var options = {new: true};
+            return self.updateAndFindMongo(query, update, options);
+        },
+
+
+        /**
+         * Using an Id, find document and add one or more items into a list of this document, 
+         * the values provided specifies the list and items, suppose relation is a obj list,
```

Comment:
use the following format will be more readable for users than literal description
```
{
...,
relations: [
    {
        type: "encloses",
        targets: ["123", "456"]
    },
    {
        type: "contains",
        targets: ["abc", "def]
    }
],
...
}
```